fmlearlydisplay/src/main/java/net/minecraftforge/fml/earlydisplay/DisplayWindow.java

```diff
@@ -402,11 +400,23 @@ public void initWindow(@Nullable String mcVersion) {
                 continue;
             }
             LOGGER.info("Trying GL version " + glVersionToTry);
+            if (showHelpLog && versidx == 0) {
+                LOGGER.info("""
+                If this message is the only thing at the bottom of your log before a crash, you probably have a driver issue.
+                
+                Possible solutions:
+                A) Make sure Minecraft is set to prefer high performance graphics in the OS and/or driver control panel
```

Comment:
I have had weird niche issues where simply setting Minecraft itself does not affect what graphics card Windows will choose to run the game with. However that is more of an issue with third-party launchers. The default Minecraft Launcher from the Microsoft Store will always request that Windows uses the high performance graphics card.

Reply:
I wasn't suggesting recommending the official launcher, I was suggesting adding that they might want to add the javaw.exe binary to their list of dGPU applications. The part about the official launcher using the dGPU is just commentary of my own experience.